### DISCUSSION TITLE
Ubuntu based gotenberg docker image

### DISCUSSION THREAD

**Discussion by gotenberg82** [gotenberg/gotenberg#596] [Q&A]
Hi, 
When I tried to convert a specific docx document to pdf using online API of gotenberg: "https://demo.gotenberg.dev/forms/libreoffice/convert" , the TOC got removed in the created pdf file, except for the heading text

Source docx file : 
[document.docx](https://github.com/gotenberg/gotenberg/files/10517845/document.docx)

When tried to convert the same document in libreoffice versions: 7.4.4 and 6.4.7 in a server with Ubuntu OS, the ToC was present in the converted pdf document

Is there a gotenberg docker image build on Ubuntu OS? or steps on how to build a gotenberg  docker image based on Ubuntu OS?

**Comment by gulien** [comment]
Hello @gotenberg82,

The current LibreOffice version is `7.0.4` in Gotenberg (I had to rollback because of stability issues). You may use Gotenberg `7.7.1`, which uses a recent version of LibreOffice, but you should use the stateless mode, i.e., `--uno-listener-restart-threshold=0`.

**Comment by gotenberg82** [comment]
Thank You **@gulien** for the suggestion.
Tried out with Gotenberg 7.7.1, now everything in the ToC is repeated 2 times in the converted pdf. screenshot of that is below:
![image](https://user-images.githubusercontent.com/123725082/215150300-d1e70b99-455e-4319-9b71-b51d2b8bea42.png)

Since the conversion of document was fine in LibreOffice installed in Ubuntu OS, Is there a Gotenberg docker image build based on Ubuntu OS?
